Doc/library/textwrap.rst

```diff
@@ -17,31 +17,63 @@ If you're just wrapping or filling one or two text strings, the convenience
 functions should be good enough; otherwise, you should use an instance of
 :class:`TextWrapper` for efficiency.
 
-.. function:: wrap(text, width=70, **kwargs)
+.. function:: wrap(text, width=70, initial_indent="", subsequent_indent="", \
+                   expand_tabs=True, replace_whitespace=True, \
+                   fix_sentence_endings=False, \
+                   break_long_words=True, drop_whitespace=True, \
+                   break_on_hyphens=True, tabsize=8, \
+                   *, max_lines=None, **kwargs)
 
    Wraps the single paragraph in *text* (a string) so every line is at most
    *width* characters long.  Returns a list of output lines, without final
-   newlines.
+   newlines.  *width* defaults to ``70``.
 
-   Optional keyword arguments correspond to the instance attributes of
-   :class:`TextWrapper`, documented below.  *width* defaults to ``70``.
+   See :class:`TextWrapper` for documentation on all additional optional
+   arguments:
 
-   See the :meth:`TextWrapper.wrap` method for additional details on how
-   :func:`wrap` behaves.
+   * :attr:`.initial_indent`
+   * :attr:`.subsequent_indent`
+   * :attr:`.expand_tabs`
+   * :attr:`.replace_whitespace`
+   * :attr:`.fix_sentence_endings`
+   * :attr:`.break_long_words`
+   * :attr:`.drop_whitespace`
+   * :attr:`.break_on_hyphens`
+   * :attr:`.tabsize`
+   * :attr:`.max_lines`
 
 
-.. function:: fill(text, width=70, **kwargs)
+.. function:: fill(text, width=70, initial_indent="", \
+                   subsequent_indent="", expand_tabs=True, \
+                   replace_whitespace=True, fix_sentence_endings=False, \
+                   break_long_words=True, drop_whitespace=True, \
+                   break_on_hyphens=True, tabsize=8, \
+                   *, max_lines=None, **kwargs)
 
-   Wraps the single paragraph in *text*, and returns a single string containing the
-   wrapped paragraph.  :func:`fill` is shorthand for  ::
+   Wraps the single paragraph in *text*, and returns a single string containing
+   the wrapped paragraph.  :func:`fill` is shorthand for ::
 
       "\n".join(wrap(text, ...))
 
-   In particular, :func:`fill` accepts exactly the same keyword arguments as
-   :func:`wrap`.
+   See :class:`TextWrapper` for documentation on all additional optional
+   arguments:
 
+   * :attr:`.initial_indent`
+   * :attr:`.subsequent_indent`
+   * :attr:`.expand_tabs`
+   * :attr:`.replace_whitespace`
+   * :attr:`.fix_sentence_endings`
+   * :attr:`.break_long_words`
+   * :attr:`.drop_whitespace`
+   * :attr:`.break_on_hyphens`
+   * :attr:`.tabsize`
+   * :attr:`.max_lines`
 
-.. function:: shorten(text, width, **kwargs)
+
+.. function:: shorten(text, width, initial_indent="", \
+                      subsequent_indent="", fix_sentence_endings=False, \
+                      break_long_words=True, break_on_hyphens=True, \
+                      *,  max_lines=None,  placeholder=' [...]', **kwargs)
```

Comment:
*max_lines* is not accepted. *subsequent_indent* should not have effect. *initial_indent* is not useful (perhaps it should be forbidden).

Reply:
Thank you, this is fixed. See d6b5fe08b70102b63c02a0861cb65587aeac6144